### ISSUE TITLE
Elastic skin: program/actions/mail/pagenav.php not working

### ISSUE THREAD

**Issue by netchild** [roundcube/roundcubemail#8798]
I tried to modify the sort order of the paging (prev and next buttons) in the run function of pagenav.php, but the elastic skin doesn't pick up my modifications.

Quick example/test (reverse the meaning of prev and next based upon the sort order):
```
        if ($cnt && ($pos = $index->exists($uid, true)) !== false) {
            if (self::sort_order() == 'ASC') {
                $prev = $pos ? $index->get_element($pos-1) : 0;
                $next = $pos < $cnt-1 ? $index->get_element($pos+1) : 0;
            } else {
                $prev = $pos < $cnt-1 ? $index->get_element($pos+1) : 0;
                $next = $pos ? $index->get_element($pos-1) : 0;
            }
            $first = $pos ? $index->get_element('FIRST') : 0;
            $last  = $pos < $cnt-1 ? $index->get_element('LAST') : 0;
        }
```

If I got the roundcube navigation logic wrong and this is the wrong place to modify the left and right buttons in the message-list, where is the correct place to modify this?

**Comment by alecpl** [comment]
Why?

**Comment by netchild** [comment]
I want to have the navigation and sort order in a way that the oldest message is at the top and the newest at the bottom, and that when I login I get presented with the last page of the messages so that the newest can be seen at the bottom. I'm used to this way over several mail clients over a time span of about 25 years. Unfortunately when I change the sort order, I get the first page of the message list with the oldest messages, and not the last page with the newest messages. To be able to implement my way, I need to find the correct place where this is done. I thought by reading the code, that this is the place which affects parts of this. By changing this navigational aspect I wanted to validate that I found the right place where I can change it to my liking. Unfortunately, at least with the default theme which I tried, this change doesn't pass through to the interface.
